compiler/cmd/check-backward-compatibility/check-backward-compatibility.go

```diff
@@ -11,13 +12,24 @@ import (
 )
 
 func main() {
-	var yamlsPath string
+	var (
+		yamlsPath    string
```

Comment:
yamlsPath --> is the yaml template
oldYamlPath --> is the entire CRD yaml
correct ?
If yes, can we rename it to existingCRDsPath ?

Reply:
Modified.